bitnami/harbor/requirements.lock

```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: postgresql
   repository: https://charts.bitnami.com/bitnami
-  version: 7.7.3
+  version: 8.x.x
```

Comment:
I think doing this the version should be a major.
Also, we should test these changes properly in our pipeline so please, open a different PR for the dependencies update.

Reply:
I already did that at https://github.com/bitnami/charts/pull/1936
Please rebase from master instead.